lib/none/none.performance.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function() {
+  const isEven = x => x % 2 === 0
+  const listEven = []
+  const listOdd = []
+  let listMixed = []
+  const length = 100000
+
+  for (let i = 0; i < length; i++) {
+    if (isEven(i)) {
+      listEven.push(i)
+      listMixed.push(i)
+    } else {
+      listOdd.push(i)
+      listMixed.unshift(i)
+    }
+  }
+
+  const sets = [
+    [isEven, listEven], // 0% match
+    [isEven, listMixed], // 50% match
+    [isEven, listOdd] // 100% match
+  ]
+
+  return {
+    argss: sets,
+    type: 'array_percent'
+  }
+}
```

Comment:
Please have a look at `performance.js` and use constants instead of numbers you have.
And also try to generate big lists with `generateArrays` or `generateObjects`

Reply:
Could not generate specified arrays with `generateArrays`, because I need three arrays with the same length, but specified items and `generateArrays` provides exactly length-specified arrays.

----------

> use constants instead of numbers you have

OK

Reply:
> provides exactly length-specified arrays.

Agree